ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -306,7 +306,6 @@ tests:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-stack-upi
 - as: baremetal-upi-ovn-ipv4-compact-f28
-  cluster: build05
```

Comment:
I think we use build05 for baremetal because build05 was the first build farm that has intranet access.
If we simply remove build05, the installation can success if the job reschedule to another build farm cluster that also has intranet access, while the installation will fail if the new build farm cluster does not have intranet access.
Refer to [doc](https://docs.ci.openshift.org/docs/how-tos/capabilities/) for how do we handle that via capabilities now.
BTW, I would suggest split the PR into several smaller one, e.g, one for remove jobs, one for merge jobs, and one more to switch to capabilities, to decrease the possibility of conflicts, and also for easier review.

Reply:
Sure, will split it later. 
I also tried to use capabilities locally, but I see some baremetal jobs are passed so I haven't submit the commit.